src/messages/manifestjson.js

```diff
@@ -455,6 +455,17 @@ export const WRONG_ICON_EXTENSION = {
   file: MANIFEST_JSON,
 };
 
+export const IGNORED_APPLICATIONS_PROPERTY = {
+  code: 'IGNORED_APPLICATIONS_PROPERTY',
+  message: i18n._(
+    'Usage of both "browser_specific_settings" and "applications"'
```

Comment:
Nitpick: it may be reasonable to use something like `"applications" property overridden by "browser_specific_settings" property` as the short message, because it should enough but also more directly describe the issue being pointed out by the warning.

Reply:
@rpl sure and done :smiley: